src/file.cpp

```diff
@@ -503,6 +503,29 @@ namespace libtorrent
 #endif
 	}
 
+	void change_directory(std::string const& f, error_code& ec)
```

Comment:
I would actually prefer if this function stays in the test util library. libtorrent should never update current working directory. But it does make sense for the tests